triton_src/max.py

```diff
@@ -0,0 +1,91 @@
+import logging
+from collections import namedtuple
+import triton
+import triton.language as tl
+
+from flag_gems import runtime
+from flag_gems.runtime import torch_device_fn
+from flag_gems.utils import dim_compress, libentry, libtuner
+from flag_gems.utils import triton_lang_extension as tle
+from flag_gems.utils.limits import get_dtype_min
+
+logger = logging.getLogger(__name__)
+
+
+@libentry()
+@triton.jit
+def max_kernel_1(
```

Comment:
Thanks for being fast-pacing, but we can now use triton jit function from src/flag_gems/ops now. See examples in 

https://github.com/FlagOpen/FlagGems/pull/817

Reply:
If this file is no longer needed, we  can delete it.